### ISSUE TITLE
在训练自己的数据的时候，训练损失在下降，但是验证的各个指标总在一个值附近徘徊，大趋势不增也不降

### ISSUE THREAD

**Issue by YOLO-study-del** [Pointcept/Pointcept#587]
我准备了类似s3dis数据集组织格式的数据集，发现训练损失在下降，但是验证的各个指标总在一个值附近徘徊，大趋势不增也不降，下面是我的结果图以及配置文件，求解答
结果图如下：

<img width="2462" height="975" alt="Image" src="https://github.com/user-attachments/assets/2dccb7e7-4ee9-49d5-8546-dab758e5f799" />
配置文件如下：
_base_ = ["../_base_/default_runtime.py"]

# misc custom setting
batch_size = 1  # bs: total bs in all gpus
num_worker = 1
mix_prob = 0.8
empty_cache = True
enable_amp = False

# model settings
model = dict(
    type="DefaultSegmentorV2",
    num_classes=6,
    backbone_out_channels=64,
    backbone=dict(
        type="PT-v3m1",
        in_channels=3,
        order=("z", "z-trans", "hilbert", "hilbert-trans"),
        stride=(2, 2, 2, 2),
        enc_depths=(2, 2, 2, 6, 2),
        enc_channels=(32, 64, 128, 256, 512),
        enc_num_head=(2, 4, 8, 16, 32),
        enc_patch_size=(1024, 1024, 1024, 1024, 1024),
        dec_depths=(2, 2, 2, 2),
        dec_channels=(64, 64, 128, 256),
        dec_num_head=(4, 4, 8, 16),
        dec_patch_size=(1024, 1024, 1024, 1024),
        mlp_ratio=4,
        qkv_bias=True,
        qk_scale=None,
        attn_drop=0.0,
        proj_drop=0.0,
        drop_path=0.3,
        shuffle_orders=True,
        pre_norm=True,
        enable_rpe=False,
        enable_flash=True,
        upcast_attention=False,
        upcast_softmax=False,
        cls_mode=False,
        pdnorm_bn=False,
        pdnorm_ln=False,
        pdnorm_decouple=True,
        pdnorm_adaptive=False,
        pdnorm_affine=True,
        pdnorm_conditions=("ScanNet", "S3DIS", "Structured3D"),
    ),
    criteria=[
        dict(type="CrossEntropyLoss", loss_weight=1.0, ignore_index=225, weight=[1.5, 1.5, 1.0, 1.5, 1.5, 1.5]),
        dict(type="LovaszLoss", mode="multiclass", loss_weight=1.0, ignore_index=225),
    ],
)

# scheduler settings
epoch = 3000
optimizer = dict(type="AdamW", lr=0.005, weight_decay=0.05)
scheduler = dict(
    type="OneCycleLR",
    max_lr=[0.005, 0.0005],
    pct_start=0.05,
    anneal_strategy="cos",
    div_factor=10.0,
    final_div_factor=1000.0,
)
param_dicts = [dict(keyword="block", lr=0.0005)]

# dataset settings
dataset_type = "S3DISDataset"
data_root = "bri_data/processed_only_bridge"

data = dict(
    num_classes=6,
    ignore_index=225,
    names=[
        "pier",
        "bent_cap",
        "deck_slab",
        "guardrail",
        "abutment",
        "support",
    ],
    train=dict(
        type=dataset_type,
        split=("Area_1","Area_2",),
        data_root=data_root,
        transform=[
            dict(type="DebugStats", name="00_raw_input",only_non_finite=True, raise_error=False),
            dict(type="CenterShift", apply_z=True),
            #dict(type="DebugStats", name="01_after_CenterShift_z",only_non_finite=True),
            dict(
                type="RandomDropout", dropout_ratio=0.2, dropout_application_ratio=0.2
            ),
            dict(type="DebugStats", name="02_after_RandomDropout",only_non_finite=True, raise_error=False),
            # dict(type="RandomRotateTargetAngle", angle=(1/2, 1, 3/2), center=[0, 0, 0], axis="z", p=0.75),
            dict(type="RandomRotate", angle=[-1, 1], axis="z", center=[0, 0, 0], p=0.5),
            #dict(type="DebugStats", name="03_after_RandomRotate_z",only_non_finite=True),
            dict(type="RandomRotate", angle=[-1 / 64, 1 / 64], axis="x", p=0.5),
            #dict(type="DebugStats", name="04_after_RandomRotate_x",only_non_finite=True),
            dict(type="RandomRotate", angle=[-1 / 64, 1 / 64], axis="y", p=0.5),
            #dict(type="DebugStats", name="05_after_RandomRotate_y",only_non_finite=True),
            dict(type="RandomScale", scale=[0.9, 1.1]),
            #dict(type="DebugStats", name="06_after_RandomScale",only_non_finite=True),
            # dict(type="RandomShift", shift=[0.2, 0.2, 0.2]),
            dict(type="RandomFlip", p=0.5),
            #dict(type="DebugStats", name="07_after_RandomFlip",only_non_finite=True),
            dict(type="RandomJitter", sigma=0.005, clip=0.02),
            #dict(type="DebugStats", name="08_after_RandomJitter",only_non_finite=True),
            # dict(type="ElasticDistortion", distortion_params=[[0.2, 0.4], [0.8, 1.6]]),
            #dict(type="ChromaticAutoContrast", p=0.2, blend_factor=None),
            #dict(type="ChromaticTranslation", p=0.95, ratio=0.05),
            #dict(type="ChromaticJitter", p=0.95, std=0.05),
            # dict(type="HueSaturationTranslation", hue_max=0.2, saturation_max=0.2),
            # dict(type="RandomColorDrop", p=0.2, color_augment=0.0),
            dict(
                type="GridSample",
                grid_size=0.04,
                hash_type="fnv",
                mode="train",
                return_grid_coord=True,
            ),
            dict(type="DebugStats", name="09_after_GridSample",only_non_finite=True, raise_error=False),
            dict(type="SphereCrop", sample_rate=0.6, mode="random"),
            dict(type="DebugStats", name="10_after_SphereCrop_rate",only_non_finite=True, raise_error=False),
            dict(type="SphereCrop", point_max=100000, mode="random"),
            dict(type="DebugStats", name="11_after_SphereCrop_max",only_non_finite=True, raise_error=False),
            dict(type="CenterShift", apply_z=False),
            dict(type="DebugStats", name="12_after_CenterShift_noz",only_non_finite=True, raise_error=False),
            #dict(type="NormalizeColor"),
            # dict(type="ShufflePoint"),
            dict(type="ToTensor"),
            dict(type="DebugStats", name="13_after_ToTensor",only_non_finite=True, raise_error=False),
            dict(
                type="Collect",
                keys=("coord", "grid_coord", "segment"),
                feat_keys=("coord",),
            ),
        ],
        test_mode=False,
    ),
    val=dict(
        type=dataset_type,
        split="Area_3",
        data_root=data_root,
        transform=[
            dict(type="CenterShift", apply_z=True),
            dict(type="Copy", keys_dict={"segment": "origin_segment"}),
            dict(
                type="GridSample",
                grid_size=0.04,
                hash_type="fnv",
                mode="train",
                return_grid_coord=True,
                return_inverse=True,
            ),
            dict(type="CenterShift", apply_z=False),
            #dict(type="NormalizeColor"),
            dict(type="ToTensor"),
            dict(
                type="Collect",
                keys=("coord", "grid_coord", "segment", "origin_segment", "inverse"),
                feat_keys=("coord",),
            ),
        ],
        test_mode=False,
    ),
    test=dict(
        type=dataset_type,
        split="Area_4",
        data_root=data_root,
        transform=[
            dict(type="CenterShift", apply_z=True),
            #dict(type="NormalizeColor"),
        ],
        test_mode=True,
        test_cfg=dict(
            voxelize=dict(
                type="GridSample",
                grid_size=0.04,
                hash_type="fnv",
                mode="test",
                return_grid_coord=True,
            ),
            crop=None,
            post_transform=[
                dict(type="CenterShift", apply_z=False),
                dict(type="ToTensor"),
                dict(
                    type="Collect",
                    keys=("coord", "grid_coord", "index"),
                    feat_keys=("coord",),
                ),
            ],
            aug_transform=[[]
                # [dict(type="RandomScale", scale=[0.9, 0.9])],
                # [dict(type="RandomScale", scale=[0.95, 0.95])],
                # [dict(type="RandomScale", scale=[1, 1])],
                # [dict(type="RandomScale", scale=[1.05, 1.05])],
                # [dict(type="RandomScale", scale=[1.1, 1.1])],
                # [
                #     dict(type="RandomScale", scale=[0.9, 0.9]),
                #     dict(type="RandomFlip", p=1),
                # ],
                # [
                #     dict(type="RandomScale", scale=[0.95, 0.95]),
                #     dict(type="RandomFlip", p=1),
                # ],
                # [
                #     dict(type="RandomScale", scale=[1, 1]),
                #     dict(type="RandomFlip", p=1),
                # ],
                # [
                #     dict(type="RandomScale", scale=[1.05, 1.05]),
                #     dict(type="RandomFlip", p=1),
                # ],
                # [
                #     dict(type="RandomScale", scale=[1.1, 1.1]),
                #     dict(type="RandomFlip", p=1),
                # ],
            ],
        ),
    ),
)
